integrations/cross_platform/yocto.rst

```diff
@@ -0,0 +1,271 @@
+  - `CMAKE_FIND_ROOT_PATH_MODE_INCLUDE <https://cmake.org/cmake/help/v3.0/variable/CMAKE_FIND_ROOT_PATH_MODE_INCLUDE.html>`_
+  - `CMAKE_FIND_ROOT_PATH_MODE_PACKAGE <https://cmake.org/cmake/help/v3.0/variable/CMAKE_FIND_ROOT_PATH_MODE_PACKAGE.html>`_
+
+You can `create your own Yocto SDKs <https://www.yoctoproject.org/docs/2.6/sdk-manual/sdk-manual.html#sdk-building-an-sdk-installer>`_
+or download and use `the prebuilt ones <http://downloads.yoctoproject.org/releases/yocto/yocto-2.6.2/toolchain/x86_64/>`_.
+
+Building Conan packages with the SDK toolchain
+----------------------------------------------
+
+After setting up your desired SDK, you can start creating Conan packages setting up the environment of the Yocto SDK and running a
+:command:`conan create` command with a suitable profile with the specific architecture of the toolchain.
+
+For example, creating packages for `arch=armv8`:
+
+The profile will be:
+
+.. code-block:: text
+   :caption: *armv8*
+
+    [settings]
+    os_build=Linux
+    arch_build=x86_64
+    os=Linux
+    arch=armv8
+    compiler=gcc
+    compiler.version=8
+    compiler.libcxx=libstdc++11
+    build_type=Release
+
+Activate the SDK environment and execute the create command.
+
+.. code-block:: bash
+
+    $ source oe-environment-setup-aarch64-poky-linux
+    $ conan create . user/channel --profile armv8
+
+This will generate the packages using the Yocto toolchain. Now you can upload the binaries to an Artifactory server so share and reuse in
+your Yocto builds.
+
+.. important::
+
+    We strongly recommend to use the Yocto's SDK toolchain to create packages as they will be built with the optimization flags suitable to
+    be deployed later to an image generated in a Yocto build.
+
+Deploying an application to a Yocto image
+=========================================
+
+Once you have created and and uploaded the Conan packages to a remote in Artifactory, you can deploy them in a Yocto build.
+
+Prepare your recipes
+--------------------
+
+It is important that in order to accomplish this, that the recipe of the application to be deployed should have a
+`deploy() method <https://docs.conan.io/en/latest/devtools/running_packages.html>`_. There you can specify the files of the application
+needed in the final image as well as any other from its dependencies (like shared libraries or assets):
+
+.. code-block:: python
+
+    def deploy(self):
+        self.copy("*", src="bin", dst="bin")
+        self.copy("*.dll", src="bin", dst="bin")
+        self.copy("*.dylib*", src="lib", dst="bin")
+        self.copy("*.so*", src="lib", dst="bin")
+        self.copy_deps("*.dll", src="bin", dst="bin")
+        self.copy_deps("*.dylib*", src="lib", dst="bin")
+        self.copy_deps("*.so*", src="lib", dst="bin")
+
+Set up the Conan layer
+----------------------
+
+We have created a [meta-conan](LINK) layer that includes all the configuration, the Conan client and a
```

Comment:
Describe what is supposed to be the status of the "yocto" folder of the user at this point: Let's assume you have your poky folder (link to some docs maybe), then you have to enable the Conan layer.... etc